bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -547,6 +548,27 @@ impl Client {
             Ok(user_profile)
         })
     }
+
+    /// Registers a notification handler on the delegate if required
+    /// the sliding sync requires to have registered m.room.member with value
+    /// $ME and m.room.power_levels to be able to intercept the events.
+    /// This function blocks execution and should be dispatched concurrently
+    pub fn register_notification_handler(&self) {
+        let delegate = Arc::clone(&self.delegate);
+        let handler = move |notification, room: SdkRoom, _| {
```

Comment:
Shouldn't there be a sliding sync view that gets you all of the events for "active" (top of the room list) rooms? And rooms where you get notified should be active by definition, no?

Reply:
Hm, maybe we can discuss in the Sync later. I'm not sure how hard it will be to take the room name from the sliding sync list.

Reply:
Otherwise... wouldnt't be possible for the SS handling to just update the SdkRoom content directly? (ofc I imagine this is something that would need to be addressed into a separate PR)

Reply:
Not easily. We store all room metadata that comes from events as (simplified) events. This is important for instance so we know when a redaction comes in, we can check if it's about the room name, and update the room data accordingly.